plugins/jdg-nightly/pom.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.radargun</groupId>
+      <artifactId>radargun-plugin-parent</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+   </parent>
+
+   <artifactId>plugin-jdg-nightly</artifactId>
+   <name>JDG nightly build plugin for RadarGun</name>
+
+   <properties>
+      <version.infinispan>${jdg-nightly.infinispan.version}</version.infinispan>
+   </properties>
+
+   <profiles>
+      <profile>
+         <id>jdg-nightly-infinispan-version-default</id>
+         <activation>
+            <property>
+               <name>!jdg-nightly.infinispan.version</name>
+            </property>
+         </activation>
+         <properties>
+            <version.infinispan>8.5.0.ER3-redhat-1</version.infinispan>
```

Comment:
Seems like this should be a snapshot.

Reply:
OK, I'll look at the versions coming out of the nightly builds and update this